technical-background.md

```diff
@@ -1,11 +1,14 @@
 # Technical Background
-## Ethereum
-Ethereum is a public, transparent, permissionless blockchain system and network of nodes, supporting account-based smart contracts, where business logic can be deployed as code to create an immutable and uncensorable contract which can hold and control the flow of value. The Ethereum mainnet went live in 2015 and is the most mature and adopted smart contract system.
 
-Using a smart contract, it is possible to use the Ethereum network as an uncensorable storage and management system.
+This section briefly covers the key technologies on which Obscuro relies.
+
+We recommend reading through the "Trusted Execution Environment" section because it introduces concepts and notations used throughout the paper.
+
+## Ethereum
+Ethereum is a public, transparent, permissionless blockchain system and network of nodes, supporting account-based smart contracts, where business logic can be deployed as code to create an immutable and uncensorable contract which can hold and control the flow of value. The Ethereum mainnet went live in 2015 and is the most mature and adopted smart contract system. Read more on the official [website](https://ethereum.org/en/).
 
 ## Trusted Execution Environment
-A TEE is a secure area of a main processor or CPU. It guarantees code and data loaded inside to be protected with respect to confidentiality and integrity as it is processed. Obscuro focuses on Intel's SGX, based on the team's 5 years of experience developing a Java-based confidential computing product with it. The TEE data can not be read or processed outside the SGX enclave, including processes running at higher privilege levels in the same host.
+A TEE is a secure area of a main processor or CPU. It guarantees code and data loaded inside to be protected with respect to confidentiality and integrity as it is processed. Obscuro focuses initially on Intel's SGX, based on the team's 5 years of experience developing a confidential computing product with it. The TEE data can not be read or processed outside the enclave, including processes running at higher privilege levels in the same host.
 
 An SGX-capable CPU has two device root keys that are fused into it by the manufacturer, the _Root Provisioning Key_ (RPK) and the _Root Sealing Key_ (RSK). The RPK is known to Intel and used to prove a CPU is genuine via remote attestation and the RSK is not known to any entity outside the CPU. These keys can be used to create other CPU specific keys. In this whitepaper we will refer to them as the _Enclave Key_ (EK).  Processes and users outside the enclave encrypt data that is only meant for the enclave using keys generated inside the enclave. When the enclave wishes to store data, it is again encrypted so that the host (the server which stores the data) is not able to see it.
```

Comment:
An SGX-capable CPU has two device root keys fused into it by the manufacturer, the _Root Provisioning Key_ (RPK) and the _Root Sealing Key_ (RSK). The RPK is known to Intel and used to prove a CPU is genuine via remote attestation, and the RSK is not known to any entity outside the CPU. These keys can create other CPU specific keys. In this whitepaper, we refer to them as the _Enclave Key_ (EK). Processes and users outside the enclave encrypt data that is only meant for the enclave using keys generated inside the enclave. When the enclave wishes to store data, it is again encrypted so that the host (the server which stores the data) cannot see it.